src/datawave_cli/dictionary_interactions.py

```diff
@@ -23,38 +22,46 @@ def __init__(self, args, log: logging.Logger = logging.getLogger('dictionary_int
     def get_pod_ip(self):
         return pods.get_specific_pod(pods.web_dictionary_info, self.namespace).pod_ip
 
-    def get_entire_dictionary(self, auths: str, data_types: str, file: str):
-        """
-        Display or save the entire dictionary of datawave, all data types.
+    def get_dictionary(self, auths: str, data_types: str, filename: str):
```

Comment:
Need to get a pull request going in the web app updating the usage to this new name.